frontend-create-library/src/app/components/list-dialog/list-dialog.component.html

```diff
@@ -2,7 +2,7 @@
   <h1 class="title-text" mat-dialog-title>{{ data.titleText }}</h1>
   <select (change)="onSurveyChange($event)" class="publications-dropdown">
     <option *ngFor="let publication of data.listItems" [value]="publication.id">
-      {{ publication.title }}  {{ i18n + "createdAt" | translate }} {{ publication.createdTime | date:'long' }} 
+      {{ publication.title }}
```

Comment:
Right now the publication title is the school name. Those other attributes are added in order to differentiate between them. Until we have a more clear spec I would keep them. 
```suggestion
      {{ publication.title }} {{ i18n + "createdAt" | translate }} {{ publication.createdTime | date:'long' }}
```